src/mappers/model.js

```diff
@@ -37,7 +37,7 @@ export const mapModelToData = (model, props) => {
     } else if (type === 'number' || type === 'slider') {
       return propValue || 0
```

Comment:
Same bug with `number` and `defaultValue = 0`
Should be :
`return propValue === 0 ? propValue : (propValue || 0)`